src/main/java/com/j256/ormlite/stmt/mapped/BaseMappedStatement.java

```diff
@@ -73,11 +73,15 @@ static void appendWhereFieldEq(DatabaseType databaseType, FieldType fieldType, S
 		sb.append("= ?");
 	}
 
-	static void appendTableName(DatabaseType databaseType, StringBuilder sb, String prefix, String tableName) {
```

Comment:
Dropping this will most probably break existing apps. Why not add another function with its own signature?

Reply:
It is internal API, I am not expecting that it is used in applications.
Anyway I restored old method.